autonomy/deploy/generators/kubernetes/templates.py

```diff
@@ -306,10 +306,10 @@
 AGENT_SECRET_TEMPLATE: str = """
 apiVersion: v1
 stringData:
-    private_key: '{private_key}'
+  private_key: '{private_key}'
 kind: Secret
 metadata:
-  annotations:
+  annotations: {{}}
```

Comment:
I don't get it.
This will simply print out 
```
metadata:
  annotations: {}
```
  
  on the build.yaml file, instead of
  
```
metadata:
  annotations:
```

Reply:
(i.e., double curly braces are the escaped version to print them in the output)

Reply:
This builds and runs however, its just your linter complaining no?

Notice, if you run a (for example) helm linter on this output it passes
```
metadata:
  annotations:
```